text/logs/0000-logging-library-sdk.md

```diff
@@ -0,0 +1,251 @@
+# Logging Library SDK Prototype Specification
+
+Initial draft specification to create prototypes of OpenTelemetry Logging
+Library SDKs.
```

Comment:
**Status**

Initial draft specification to create prototypes of OpenTelemetry Logging
Library SDKs.

**Prototypes**

* x
* y
* z

Reply:
I added "Status" to be explicit.

Will add a list of prototypes later when we have them.